src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -1241,14 +1283,14 @@ object LazyList extends SeqFactory[LazyList] {
     * @param elem the element composing the resulting LazyList
     * @return the LazyList containing an infinite number of elem
     */
-  def continually[A](elem: => A): LazyList[A] = newLL(sCons(elem, continually(elem)))
+  def continually[A](elem: => A): LazyList[A] = newLL(eagerCons(elem, continually(elem)))
 
   override def fill[A](n: Int)(elem: => A): LazyList[A] =
-    if (n > 0) newLL(sCons(elem, fill(n - 1)(elem))) else empty
+    if (n > 0) newLL(eagerCons(elem, LazyList.fill(n - 1)(elem))) else empty
```

Comment:
out of curiosity, why `LazyList.`?

Reply:
I guess because its parameters are unrelated to LazyList, though the return type needs to work out, so we'd catch it if it picked up some other method. I also left the prefix for `LazyList.from` / `LazyList.continually` calls.